2020-11-01-qubes-4-0-4-rc1.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Qubes OS 4.0.4-rc1 has been released!"
+categories: releases
+download_url: /downloads/
+author: Andrew David Wong
+---
+
+We're pleased to announce the first release candidate for Qubes OS
+4.0.4.
+
+Qubes OS 4.0.4-rc1 includes many updates over the initial 4.0 release,
+in particular:
+
+- All 4.0 dom0 updates to date
+- Fedora 32 TemplateVM
+- Debian 10 TemplateVM
+- Whonix 15 Gateway and Workstation TemplateVMs
+- Linux kernel 4.19 by default
+
+Qubes 4.0.4-rc1 is available on the [downloads] page.
+
+
+What is a point release?
+------------------------
+
+A point release does not designate a separate, new version of Qubes OS.
+Rather, it designates its respective major or minor release (in this
+case, 4.0) inclusive of all updates up to a certain point. Installing
+Qubes 4.0 and fully updating it results in the same system as installing
+Qubes 4.0.4.
+
+
+What should I do?
+-----------------
+
+If you installed Qubes 4.0, 4.0.1, 4.0.2, or 4.0.3 and have [fully
+updated], then your system is already equivalent to a Qubes 4.0.4
+installation. No further action is required.
+
+Regardless of your current OS, if you wish to install (or reinstall)
+Qubes 4.0 for any reason, then the 4.0.4 ISO makes this more convenient
+and secure, since it bundles all Qubes 4.0 updates to date. Please see
+the [installation guide] for detailed instructions.
+
+If you're willing to [test] this release candidate, you can help to
+improve the stable release by [reporting any bugs you encounter].
+
+
+Release candidate planning
+--------------------------
+
+If no major issues are discovered in 4.0.4-rc1, we expect to announce
+the stable release of 4.0.4 soon.
```

Comment:
We use "soon" quite freely. I think more specific term could be used here, like "couple of weeks", or even "2-3 weeks".

Reply:
Updated!